crates/fluvio-cli/src/client/produce/mod.rs

```diff
@@ -199,47 +199,30 @@ mod cmd {
             fluvio: &Fluvio,
         ) -> Result<()> {
             init_monitoring(fluvio.metrics());
-            let config_builder = if self.interactive_mode() {
-                TopicProducerConfigBuilder::default().linger(std::time::Duration::from_millis(10))
-            } else {
-                Default::default()
+            let mut config_builder = TopicProducerConfigBuilder::default();
+            if self.interactive_mode() {
```

Comment:
do we still need interactive mode?

Reply:
Do you mean have different linger on interactive mode? Or about interactive mode itself?

Reply:
I meat interactive mode.   Doesn't seem like nobody uses and not sure why it set to linger non zero

Reply:
probably for interactive mode we should use the default now that it is 0.

Reply:
Ok, I removed this condition as the default is 0 now